persistence/block.go

```diff
@@ -80,13 +83,21 @@ func (p *PostgresContext) prepareBlock(proposerAddr, quorumCert []byte) (*coreTy
 		txs[i] = txResult.GetTx()
 	}
 
+	// Retrieve the network id from the config
+	networkId := viper.GetString("network_id")
+
+	// Get the current timestamp
+	timestamp := timestamppb.Now()
```

Comment:
```suggestion
	// TECHDEBT: This will lead to different timestamp in each node's block store because `prepareBlock` is called locally. Needs to be revisisted and decided on a proper implementation.
	timestamp := timestamppb.Now()
```